server/app.js

```diff
@@ -10,18 +10,26 @@ app.use(middleware.morgan('dev'));
 app.use(middleware.cookieParser());
 app.use(middleware.bodyParser.urlencoded({extended: false}));
 app.use(middleware.bodyParser.json());
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'ejs');
 
 app.use(middleware.auth.session);
 app.use(middleware.passport.initialize());
 app.use(middleware.passport.session());
-app.use(middleware.flash());
 
-app.use(express.static(path.join(__dirname, '../public')));
+// Serve static files
+app.get('*/bundle.js', (req, res) => {
```

Comment:
This seems weird. Why do you need to do this? Shouldn't this be handled by express.static?